### ISSUE TITLE
Dev server

### ISSUE THREAD

**Issue by nmorduch** [newamericafoundation/newamerica-cms#1410]
I would like, in the long run, to have the staging branch consistently be for staging changes to production, and to have the ability to test feature branches on the web. The question is whether there is a fairly simple and inexpensive way to do this. (e.g. a duplicate of the staging server that's called dev and that we can easily push a given feature branch to)

**Comment by nmorduch** [comment]
Will reassess after #1476
